src/components/AttachmentPicker/index.native.js

```diff
@@ -227,7 +233,7 @@ class AttachmentPicker extends Component {
                 .then(this.pickAttachment)
                 .catch(console.error)
                 .finally(() => delete this.onModalHide),
-            10,
+            25,
```

Comment:
Unfortunately this is still needed even after the major version update for both the image-picker and the document-picker